.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,7 +17,7 @@ A clear and concise description of what you expected to happen.
  - HDF5 version (if building from a maintenance branch, please include the commit hash)
  - OS and version
  - Compiler and version
- - Build system (e.g. CMake, Autotools) and version
+ - Build system (e.g. CMake) and version
```

Comment:
This could probably be tweaked. There's only one build system, but I'd be interested in knowing what CMake was targeting: e.g., Xcode vs Makefiles, etc.

Reply:
Fixed